### DISCUSSION TITLE
How the hell I make new repo with this UI?

### DISCUSSION THREAD

**Discussion by rust2** [community/community#65969] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I hover almost every button in this UI it becomes unpressable. How am I supposed to make new repo now?

https://github.com/community/community/assets/43823272/561f837f-a156-4cac-a008-48b6c33e0a51

**Comment by rust2** [comment]
oh, I can actually press that button if I aim it precisely...
![image](https://github.com/community/community/assets/43823272/39075763-c485-413e-9ba1-c0def7b38cdb)

**Comment by lomelo-x** [comment]
Drop down the menu in **red** then select "new repository" in **green**

![image](https://github.com/community/community/assets/91914856/9d4fc0e4-19b6-41a8-baa0-efbc439d6862)

**Reply by mehulrajdev** [reply]
Way to make new one

**Reply by rust2** [reply]
Did you see that when I hover that "plus", it gets covered by tooltip?

**Comment by lab596** [comment]
Hey, I suggest:

1. Clear Browser Cache:
Sometimes, cached data can cause UI issues. Clear your browser's cache and reload the GitHub page to see if the problem persists.

or 

2. Try a Different Browser:
Test the GitHub page in a different web browser to determine if the issue is browser-specific. It could be related to a browser extension or a browser compatibility problem.

**Reply by rust2** [reply]
I use Chrome 86.0.4240.75 (pretty old bruh) in incognito mode. Newer version just kills my laptop

**Reply by rust2** [reply]
haha
![image](https://github.com/community/community/assets/43823272/1dc29e49-2857-4ad2-a159-b8be07a481b2)

**Reply by lab596** [reply]
> I use Chrome 86.0.4240.75 (pretty old bruh) in incognito mode. Newer version just kills my laptop

Yeah, that is most likely causing the problem
